ckanext/dcatapchharvest/profiles.py

```diff
@@ -748,18 +797,26 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
 
             g.add((dataset_ref, DCAT.distribution, distribution))
             g.add((distribution, RDF.type, DCAT.Distribution))
-
+            
             #  Simple values
             items = [
                 ('status', ADMS.status, None, Literal),
-                ('rights', DCT.rights, None, Literal),
                 ('coverage', DCT.coverage, None, Literal),
-                ('license', DCT.license, None, Literal),
                 ('identifier', DCT.identifier, None, Literal),
                 ('media_type', DCAT.mediaType, None, Literal),
                 ('spatial', DCT.spatial, None, Literal),
             ]
 
+            resource_rights = resource_dict.get('rights')
+            rights_uri = URIRef(resource_rights)
```

Comment:
This doesn't work, because the value of `resource_dict.get('rights')` is a string, e.g. "NonCommercialAllowed-CommercialAllowed-ReferenceRequired", not a URI.